share/completions/castnow.fish

```diff
@@ -0,0 +1,30 @@
+set -l __fish_castnow_keys "space\tToggle\ between\ play\ and\ pause m\tToggle\ mute t\tToggle\ subtitles up\tVolume\ up down\tVolume\ down left\tSeek\ backward right\tSeek\ forward n\tNext\ in\ playlist s\tStop\ playback quit\tQuit"
```

Comment:
![castnow_command_completion](https://cloud.githubusercontent.com/assets/19509728/22115033/bbec1830-de6b-11e6-8d89-b00efab6e538.png)

I sure did test it. I can remove the quotes if you prefer it without them.